jetty-core/jetty-deploy/src/main/config/etc/jetty-deployment-scanner.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<!-- =============================================================== -->
+<!-- Create Default Deployment Provider                              -->
+<!--                                                                 -->
+<!-- The Default Deployment Provider is responsible for monitoring   -->
+<!-- provided directories for web applications and hot deploying     -->
+<!-- through the DeploymentManager.                                  -->
+<!-- =============================================================== -->
+<Configure id="deploymentScanner" class="org.eclipse.jetty.deploy.DeploymentScanner">
```

Comment:
By making the Scanner a top level `Configure` object, then it will be directly started by `XmlConfiguration`.   However, it also is set as a beam on the deploymentManager, so it now has two paths to start it.

I think the structure is going in two ways.   By java reference, `DeploymentScanner` has-a `ContextHandlerDeployer` (deploymentManager), but it is the `ContextHandlerDeployer` that has the `DeploymentScanner` as a bean! 

We need to decide who is the boss, because the order of starting these two classes is important.    Either the scanner is the boss, is added to the server as a bean and has-a deployer to do the deployment;  OR the deployer is the boss, is added to the server as a bean and has-a scanner added to it as a bean.